package-lock.json

```diff
@@ -2191,14 +2191,6 @@
         "@types/yargs-parser": "*"
       }
     },
-    "node_modules/@jest/core/node_modules/ansi-regex": {
```

Comment:
Isn't the `package-lock` auto-generated? how did this fix the vulnerabilities? Probably I'm missing something :thinking: